imod/prepare/topsystem/allocation.py

```diff
@@ -314,6 +314,15 @@ def _enforce_layered_top(top: GridDataArray, bottom: GridDataArray):
         return create_layered_top(bottom, top)
 
 
+def _is_above_lower_bound(bottom_elevation: GridDataArray, top_layered: GridDataArray):
```

Comment:
I find this name confusing. What is the lower bound? And what is above it?

Reply:
I added documentation, and also made a distinction with ``is_below_upper_bound``